hash

```diff
@@ -46,16 +51,16 @@ sparkSubmit() {
 # https://issues.apache.org/jira/browse/SPARK-16784
 # spark.executor.extraJavaOptions=-Dlog4j.configuration=log4j-spark.properties
 
-source "$current_dir/scripts/common_params.sh"
+source "${current_dir}/scripts/common_params.sh"
 
 sparkSubmit \
-  --jars "${deps_jar}" \
   --class "${app_class}" \
   --master "${MASTER:=local[*]}" \
   --name "${app_name}" \
+  --jars "${deps_jar}" \
   --conf "spark.executor.memory=4g" \
   --conf "spark.default.parallelism=8" \
   --driver-java-options "-Dlog4j.configuration=jar:file:${jar}!/log4j.properties" \
   "${jar}" \
-  "${COMMAND_PARAMS[@]}"
+  "$@"
```

Comment:
we need COMMAND_PARAMS to read from env variables

Reply:
Could you elaborate on it plz? Not sure I understood what you mean. Some links to code/docs would be appreciated

Reply:
This script includes https://github.com/src-d/gemini/blob/master/scripts/common_params.sh
That one reads env variables and sets flags.